src/pyspiffe/svid/exceptions.py

```diff
@@ -0,0 +1,95 @@
+"""
+This module defines SVID exceptions.
+"""
+
+from pyspiffe.exceptions import PySpiffeError
+
+INVALID_VALUE_ERROR = '{} is not supported.'
+"""str: not supported error message."""
+
+
+class JwtSvidError(PySpiffeError):
+    """Top level exception for JwtSvid module.
+
+    Attributes:
+        message: message describing the error.
+
```

Comment:
nit: Extra newline (a few instances of this)

Reply:
Done! Thanks.